auction-app-frontend/src/components/ProductBidsItem/ProductBidsItem.jsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from "src/components";
+
+import { BUTTON_VARIANTS } from "src/constants";
+
+import "./style.scss"; 
+
+const ProductBidsItem = ({ imgSrc, title, timeLeft, yourPrice, highestBid, noBids, buttonLabel, onButtonClick, highestBidder }) => {
```

Comment:
bidPrice or just price is better naming than yourPrice 
yourPrice can be confusing for others